src/synchronizeContacts.js

```diff
@@ -159,6 +163,14 @@ const synchronizeContacts = async (
             contactAccountId
           )
           result.google.created++
+        } else if (shouldDeleteOnGoogle(cozyContact)) {
+          const { id: resourceName } = cozyContact.cozyMetadata.sync[
+            contactAccountId
+          ]
+          await googleUtils.deleteContact(resourceName)
+          result.google.deleted++
+          await cozyUtils.client.destroy(mergedContact)
+          result.cozy.deleted++
```

Comment:
I'm not sure we really want to increment this one

Reply:
I'm not sure either haha 😁 . Since we do permanently delete a contact I thought we should log it, even though the contact was already trashed… but if you think it's better not to, I don't mind :)

Reply:
I'm fine with this, it gives us better feedback during dev